### DISCUSSION TITLE
academo.org/.../orbit-simulator & Edward Ball

### DISCUSSION THREAD

**Discussion by WilliamEKimberly** [community/community#21501] [Other Feature Feedback, Questions, & Ideas]
<p>Hi-  I believe Edward Ball is the author of the orbit-simulator program on <a href="http://academo.org" rel="nofollow noopener">academo.org</a>.  I would like to get in touch with him to use his work in another context. Could somebody help me contact him (email is fine). </p>
<p>Thanks, Bill Kimberly</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/williamekimberly">@williamekimberly</a>!</p>
<p>Thanks for being part of the GitHub Community Forum! We’re glad you’re here.</p>
<p>I wanted to jump in here, and mention that if you don’t get the help you’re looking for from this particular community, you might want to try getting help somewhere that focuses on <a href="https://academo.org/contact/" rel="nofollow noopener">academo creators</a>. It’s definitely possible another GitHub user might have run into this same issue and can help, but the GitHub Community Forum focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this forum may not be the right place for this particular topic.</p>
